src/gui/presets.c

```diff
@@ -1241,36 +1241,22 @@ static void dt_gui_presets_popup_menu_show_internal(dt_dev_operation_t op, int32
                   MIN(bl_params_size, sizeof(dt_develop_blend_params_t))))
       isdefault = TRUE;
 
+    gchar *label;
+    if(isdefault)
+      label = g_strdup_printf("%s %s", name, _("(default)"));
+    else
+      label = g_strdup_printf("%s", name);
+    mi = gtk_menu_item_new_with_label(label);
+    g_free(label);
+
     if(module
        && !memcmp(params, op_params, MIN(op_params_size, params_size))
        && !memcmp(bl_params, blendop_params, MIN(bl_params_size, sizeof(dt_develop_blend_params_t)))
        && module->enabled == enabled)
     {
       active_preset = cnt;
       writeprotect = sqlite3_column_int(stmt, 2);
-      char *markup;
-      mi = gtk_menu_item_new_with_label("");
-      if(isdefault)
-      {
-        markup = g_markup_printf_escaped("<span weight=\"bold\">%s %s</span>", name, _("(default)"));
```

Comment:
instead of  `<span weight=...> </span>` have you tried `<b> </b>` just in case it would work on Windows.

Reply:
@TurboGit - I already asked that and it desn't work on windows.